namespaces/live.cloud-platform.service.justice.gov.uk/pre-sentence-service-prototype/resources/serviceaccount.tf

```
@@ -0,0 +1,8 @@
+module "serviceaccount" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-serviceaccount?ref=0.8.1"
```

The latest version of this module is 0.8.2
```suggestion
  source = "github.com/ministryofjustice/cloud-platform-terraform-serviceaccount?ref=0.8.2"
```